lib/chef/knife/bootstrap.rb

```diff
@@ -97,6 +97,16 @@ class Bootstrap < Knife
         :description => "The proxy server for the node being bootstrapped",
         :proc => Proc.new { |p| Chef::Config[:knife][:bootstrap_proxy] = p }
 
+      option :bootstrap_proxy_user,
+        :long => "--bootstrap-proxy-user PROXY_USER",
+        :description => "The proxy authentication username for the node being bootstrapped",
+        :proc => Proc.new { |p| Chef::Config[:knife][:bootstrap_proxy_user] = p }
```

Comment:
these :proc lines should both be able to get deleted.  a long time ago we obsoleted this way of merging config, but never cleaned up the code.